app/javascript/packs/application.tsx

```diff
@@ -112,10 +118,21 @@ initReact({
   'mentoring-queue': (data: any) => (
     <Mentoring.Queue request={data.request} sortOptions={data.sort_options} />
   ),
-  'mentoring-mentoring-panel-list': (data: any) => (
-    <Mentoring.MentoringPanelList
+  'mentoring-mentor-discussion': (data: any) => (
+    <Mentoring.MentorDiscussion
       discussionId={data.discussion_id}
-      iterationIdx={data.iteration_idx}
+      student={
+        (camelizeKeys(data.student) as unknown) as MentorDiscussionStudent
```

Comment:
We have quite some `(camelizeKeys(X) as unknown) as Y` calls in our code base now. Maybe it would be nice to add a function that would allow us to do: `camelizeKeysAs<Y>(x)`?

Reply:
Sounds good! I will write that function.